static/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Critter</title>
+    <link rel="stylesheet" href="/static/index.css">
+</head>
+<body>
+    <header>
+        <nav class="header-flex">
+            <img src="" alt="profile picture">
+            <h1>Home</h1>
+            <form action="post">
```

Comment:
If this is your main "submit crit" box, you probably don't want it in your header. This feels like it belongs down inside your timeline. I'd maybe expect a search box in the primary header/navigation section, but not the "write a crit" form.